### DISCUSSION TITLE
Can I use block element inside another nested block?

### DISCUSSION THREAD

**Discussion by BlackRabbitt** [getbem/getbem.github.io#233] [Q&A]
considering the example below, **is it against BEM to use block-element (`i-form__input1`) inside another nested block `n-block`?**

`<div class="i-form">`
`.....<div class="n-block">`
`..........<div class="i-form__input1">`
`..........</div>`
`.....</div>`
`</div>`

**Comment by dotnetCarpenter** [comment]
Should be fine according to https://github.com/getbem/getbem.github.io/discussions/225#discussioncomment-3667248
